Cargo.toml

```diff
@@ -10,9 +10,16 @@ path = "src/bin/ldns.rs"
 
 [dependencies]
 clap = { version = "4.3.4", features = ["derive"] }
-domain = "0.10.1"
+domain = { version = "0.10.3", git = "https://github.com/NLnetLabs/domain.git", features = [
+    "zonefile",
+    "bytes",
+    "unstable-validate",
+] }
 lexopt = "0.3.0"
 
 # for implementation of nsec3 hash until domain has it stabilized
-octseq = { version = "0.5.1", features = ["std"] }
+octseq = { version = "0.5.2", features = ["std"] }
```

Comment:
Will this be an issue when merging the deletion of these lines with the nsec3 pr #6?

Reply:
Possibly 🤷🏻 I suppose we'll figure it out once we get there